src/compat/compat.h

```diff
@@ -607,6 +607,25 @@ pg_strtoint64(const char *str)
 							  freezeLimit,                                                         \
 							  multiXactCutoff);                                                    \
 	} while (0)
+#else
+#define vacuum_set_xid_limits_compat(rel,                                                          \
+									 freeze_min_age,                                               \
+									 freeze_table_age,                                             \
+									 multixact_freeze_min_age,                                     \
+									 multixact_freeze_table_age,                                   \
+									 oldestXmin,                                                   \
+									 freezeLimit,                                                  \
+									 multiXactCutoff)                                              \
+	do                                                                                             \
+	{                                                                                              \
+		VacuumParams params;                                                                       \
+		memset(&params, 0, sizeof(VacuumParams));                                                  \
```

Comment:
I think the `params` should probably be initialized using the `_age` parameters being passed in. (Looks like the only caller we have sets all these to zero, so it won't cause problems yet.)

Reply:
I have updated the patch. The `vacuum_get_cutoffs` uses only the `_age` members, so only they are initialized now.